### ISSUE TITLE
rotated maps

### ISSUE THREAD

**Issue by antoniusz** [haubourg/EasyCustomLabeling#8]
Hi,
The line/s, connecting the labels, doesn't conform to the rotation of the canvas or of the map in the composer.

See attachment.

![rotated_map_with_labels](https://cloud.githubusercontent.com/assets/10415477/11366391/a86102e2-92b5-11e5-80b3-28a6a314a3b2.png)

**Comment by haubourg** [comment]
Hi, thanks for raising this. 
Current code uses map coordinates to compute labels. Handling this would require additional signal/slots to recompute them on the fly with every rotation modification. 
What is your need? Do you assume that labels should stick to their map coordinates when rotating, or do  you assume  that labels should keep flat ends on screen, which need to reposition labels on the fly?
I probably wont fix the plugin (but anyone can submit pull request, I will happily merge them), but I will raise that in https://github.com/qgis/QGIS-Enhancement-Proposals/issues/47 where we are currently discussing native labeling improvements. 
Régis

**Comment by antoniusz** [comment]
Hi,
My problem is the connector line. It doesn't rotate with the label. On the sample, the doted line - grid -  originally (on map canvas) was vertical. As you can see the indicated line (shorter part of the connector line) was made with unrotated map canvas, but after rotation in composer remained so.
I've tried to rotate the map canvas, but then i hadn't been able to move the labels.

**Comment by haubourg** [comment]
Right, so you need labels connectors to be recomputed on when map rotates. Adding this in QEP 47 as a requirement. 
Refactoring Easylabeling is a big task and I have no free time by now. I prefer funding core features. If anyone has some time to connect canvas rotation events to the connector computing part, feel free!

**Comment by haubourg** [comment]
Hum, two other points:
- If you need to modify a connector, just edit it with node tool, it is a line layer (!)
- It seems QGIS has a bug with moving manually labels and map rotation. No way to grab it on screen once rotated..
